egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1i_dp.sh

```diff
@@ -0,0 +1,295 @@
+#!/bin/bash
+# Copyright 2016  University of Chinese Academy of Sciences (Author: Cheng Gaofeng)
+# Apache 2.0
+
+# same as 1i but with frame level dropout
+# (num-params 1g:21309812 1i: 43447156)
+# results on sdm1 using ihm ali
+#System            tdnn_lstm1i_sp_bi_ihmali_ld5
+#WER on dev        37.6            36.7
+#WER on eval       40.9            39.9
+#Final train prob      -0.114135   -0.118
+#Final valid prob      -0.245208   -0.246
+#Final train prob (xent)      -1.47648  -1.54
+#Final valid prob (xent)      -2.16365  -2.10
+
+
+set -e -o pipefail
+
+# First the options that are passed through to run_ivector_common.sh
+# (some of which are also used in this script directly).
+stage=0
+mic=ihm
+nj=30
+min_seg_len=1.55
+use_ihm_ali=false
+train_set=train_cleaned
+gmm=tri3_cleaned  # the gmm for the target data
+ihm_gmm=tri3  # the gmm for the IHM system (if --use-ihm-ali true).
+num_threads_ubm=32
+nnet3_affix=_cleaned  # cleanup affix for nnet3 and chain dirs, e.g. _cleaned
+dropout_schedule='0,0@0.20,0.5@0.50,0@0.50,0'
```

Comment:
This will now crash with the new change. But it seems like in some cases this step change is required. So should I change the script to accept the step changes and retain the order?